.github/scripts/run-fabric-tests.sh

```diff
@@ -122,30 +117,28 @@ docker ps -a
 # now we need to run some benchmarks
 popd
 
-# benchmark chaincodes whose ccn is: fabcar, marbles, simple, fixed-asset
+# benchmark chaincodes whose ccn is: fabcar, marbles, simple, fixed-asset, smallbank
 
 echo "---- Running fabcar benchmarks"
 npx caliper launch manager --caliper-workspace ./ --caliper-networkconfig networks/fabric/test-network.yaml --caliper-benchconfig benchmarks/samples/fabric/fabcar/config.yaml --caliper-flow-only-test --caliper-fabric-gateway-enabled
 
-# only javascript works, go doesn't and java not available.
-# TODO: Update once go works
-if [ "$CC_SRC_LANGUAGE" = "javascript" ]; then
+# only javascript ang go work, java not available.
+if [ "$CC_SRC_LANGUAGE" != "java" ]; then
   echo "---- Running marbles benchmarks"
   npx caliper launch manager --caliper-workspace ./ --caliper-networkconfig networks/fabric/test-network.yaml --caliper-benchconfig benchmarks/samples/fabric/marbles/config.yaml --caliper-flow-only-test --caliper-fabric-gateway-enabled
 fi
 
-# only javascript works, go doesn't and java not available.
-# TODO: Update once go works
-if [ "$CC_SRC_LANGUAGE" = "javascript" ]; then
+# only javascript ang go work, java not available.
+if [ "$CC_SRC_LANGUAGE" != "java" ]; then
   echo "---- Running simple benchmarks"
   npx caliper launch manager --caliper-workspace ./ --caliper-networkconfig networks/fabric/test-network.yaml --caliper-benchconfig benchmarks/scenario/simple/config.yaml --caliper-flow-only-test --caliper-fabric-gateway-enabled
 fi
 
-# TODO: enable smallbank once go chaincode is fixed
-# if [ "$CC_SRC_LANGUAGE" = "go"]; then
-#   echo "---- Running smallbank benchmarks"
-#   npx caliper launch manager --caliper-workspace ./ --caliper-networkconfig networks/fabric/test-network.yaml --caliper-benchconfig benchmarks/scenario/smallbank/config.yaml --caliper-flow-only-test --caliper-fabric-gateway-enabled
-# fi
+# only go works, javascript and java not available.
+if [ "$CC_SRC_LANGUAGE" = "go"]; then
```

Comment:
missing a space after "go" so you get the error output in the build
.github/scripts/run-fabric-tests.sh: line 138: [: missing `]'